modules/tinymce/src/themes/silver/main/ts/ui/icons/Icons.ts

```diff
@@ -17,8 +19,30 @@ const getOr = (name: string, icons: IconProvider, fallback: Optional<string>): s
 
 const getFirst = (names: string[], icons: IconProvider): string => Arr.findMap(names, (name) => Optional.from(icons()[name.toLowerCase()])).getOrThunk(() => defaultIcon(icons));
 
+const render = (tagName: string, iconHtml: string, classes: string[], behaviours: Optional<Behaviour.AlloyBehaviourRecord>): SimpleOrSketchSpec => {
+  const iconSpec = {
+    dom: {
+      tag: tagName,
+      innerHtml: iconHtml,
+      classes
+    },
+    events: AlloyEvents.derive([
+      AlloyEvents.runOnAttached((comp) => {
+        // set focusable=false on SVGs to prevent IE 11 from focusing the toolbar when tabbing into the editor
+        SelectorFind.child(comp.element, 'svg').each((svg) => Attribute.set(svg, 'focusable', 'false'));
+      })
+    ])
+  };
+
+  return behaviours.fold(
+    Fun.constant(iconSpec),
+    (behaviour) => ({ ...iconSpec, behaviour })
```

Comment:
This is failing tests, as this needs to be `behaviours`, not `behaviour` (it's missing the trailing `s`).

Reply:
Fixed.